### PR TITLE
feat: Tracing instrument in API

### PR DESCRIPTION
**Pull request by frondeus**
Yay, no more `tracing::span_info!`!

Did four things:
* Enabled tracing for graphQL API (feature flag in dependency lol)
* Removed `tracing::span_info!` and `async move` and `.instrument()` cluttering bullshit
* Added `tracing::instrument` attribute instead.
* Formated code with `rustfmt`

### CODE REVIEW

**Comment by smores56** [comment]
This is really beautiful.

File: crates/api/src/schema/query.rs [hunk 1]
```diff
@@ -48,98 +47,88 @@ impl Schema {
...
     /// Schema is following semantic versioning, querying for "2.1.0" will return "2.1.1" if exist,
     /// querying for "=2.1.0" will return "2.1.0" if exist
+    #[tracing::instrument(skip(self, context), fields(id))]
     async fn definition(&self, context: &Context<'_>, version: String) -> FieldResult<Definition> {
-        let span = tracing::info_span!("query_definition", ?self.id, ?version);
-        async move {
-            let id = self.id.to_string();
-            let mut conn = context.data_unchecked::<SchemaRegistryPool>().get().await?;
+        let id = self.id.to_string();
+        tracing::Span::current().record("id", &id.as_str());
```
Comment (frondeus) [inline crates/api/src/schema/query.rs:53]: For unknown reason i couldn't do `fields(self.id = ?self.id)` even tho documentation says I should (https://docs.rs/tracing/0.1.25/tracing/attr.instrument.html#examples-2)... So this is way around it.

Comment (smores56) [reply crates/api/src/schema/query.rs:53]: Can you leave a comment describing such?

Comment (frondeus) [reply crates/api/src/schema/query.rs:53]: sure i will asap

**Review by smores56 — APPROVED** [review]

**Review by esavier — APPROVED** [review]

**Pull request merged**